### PR TITLE
Soc 3586 selenium

### PR DESCRIPTION
**Pull request by JurasJuras**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java [hunk 1]
```diff
@@ -2,107 +2,141 @@
...
+    driver.findElement(By.className("discussion-standalone-editor-textarea")).sendKeys(contentText);
+    clickSaveButton();
+  }
+
+  public void verifyContentNevTextInGuidelines(String content) {
+    wait.forTextInElement(contentText, content);
+  }
+
+  public void addNewTextToGuidelines() {
+    String guidelinesText = TextGenerator.createUniqueText();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java]: MAJOR Rename "guidelinesText" which hides the field declared at line 42. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by JurasJuras** [comment]
@mleczey

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java [hunk 2]
```diff
@@ -2,107 +2,141 @@
...
+    this.guidelinesText.sendKeys(contentText);
+    clickSaveButton();
+  }
+
+  public void verifyContentNevTextInGuidelines(String content) {
+    wait.forTextInElement(contentText, content);
+  }
+
+  public void addNewTextToGuidelines() {
+    String guidelinesText = TextGenerator.createUniqueText();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/GuidelinesPage.java]: MAJOR Rename "guidelinesText" which hides the field declared at line 42. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**